app/components/common/CorneredContainer.js

```diff
@@ -60,18 +60,19 @@ type Props = {
   header: string,
   headerIcon?: Object,
   subHeader?: string,
+  isErrorModal?: boolean,
   useEmptyWrap: boolean
 };
 
 class CorneredContainer extends PureComponent<Props> {
   render() {
-    const { children, width, height, header, headerIcon, subHeader, useEmptyWrap } = this.props;
+    const { children, width, height, header, headerIcon, subHeader, isErrorModal, useEmptyWrap } = this.props;
 
     return useEmptyWrap ? (
       <DivWrapper width={width} height={height}>
         <HeaderWrapper>
           {headerIcon && <HeaderIcon src={headerIcon} />}
-          <Header>{header}</Header>
+          <Header color={isErrorModal ? smColors.orange : headerColor}>{header}</Header>
```

Comment:
static defaultProps = {
    isErrorModal: false
};
or
static defaultProps = {
    headerColor: color1
};
second is even better - more generic

Reply:
fixed